cli/module_build.go

```diff
@@ -44,3 +104,136 @@ func ModuleBuildLocalAction(c *cli.Context) error {
 	infof(c.App.Writer, "Completed build")
 	return nil
 }
+
+// ModuleBuildListAction lists the module's build jobs.
+func ModuleBuildListAction(c *cli.Context) error {
+	manifestPath := c.String(moduleFlagPath)
+	manifest, err := loadManifest(manifestPath)
+	if err != nil {
+		return err
+	}
+	var numberOfJobsToReturn *int32
+	if c.IsSet(moduleBuildFlagNumber) {
+		number := int32(c.Int(moduleBuildFlagNumber))
+		numberOfJobsToReturn = &number
+	}
+	var buildIDFilter *string
+	if c.IsSet(moduleBuildFlagBuildID) {
+		filter := c.String(moduleBuildFlagBuildID)
+		buildIDFilter = &filter
+	}
+	client, err := newViamClient(c)
+	if err != nil {
+		return err
+	}
+	moduleID, err := parseModuleID(manifest.ModuleID)
+	if err != nil {
+		return err
+	}
+	jobs, err := client.listModuleBuildJobs(moduleID, numberOfJobsToReturn, buildIDFilter)
+	if err != nil {
+		return err
+	}
+	// table format rules:
+	// minwidth, tabwidth, padding int, padchar byte, flags uint
+	w := tabwriter.NewWriter(c.App.Writer, 5, 4, 1, ' ', 0)
+	tableFormat := "%s\t%s\t%s\t%s\t%s\n"
+	fmt.Fprintf(w, tableFormat, "ID", "PLATFORM", "STATUS", "VERSION", "TIME")
+	for _, job := range jobs.Jobs {
+		fmt.Fprintf(w,
+			tableFormat,
+			job.BuildId,
+			job.Platform,
+			jobStatusFromProto(job.Status),
+			job.Version,
+			job.StartTime.AsTime().Format(time.RFC3339))
+	}
+	//nolint: errcheck,gosec
```

Comment:
The table does not print to stdout until it is flushed (otherwise there could be a row of length greater than anything seen before and it could not fix previous rows.)

Will add a comment similar to that.

Reply:
Gotcha thanks!